install.sh

```diff
@@ -13,14 +13,38 @@ if ! [ -d "$EIKAIWA_BASEDIR/eikaiwa_content" ]; then
     exit 1
 fi
 
-mkdir -p ~/.config/nixpkgs
-curl -L https://github.com/iknow/nix-darwin-template/archive/master.zip | tar --strip-components 1 -C ~/.config/nixpkgs -x
+if ! [ "$SHELL" = "/bin/zsh" ]; then
+    echo "Run this script under Apple's default /bin/zsh shell"
+    exit 1
+fi
 
 # Authenticate to sudo: nix installers are going to need it, but we're detaching
 # them from the terminal
 echo "Obtaining sudo:"
 sudo echo "Obtained!"
 
+# If MacOS 10.15
+darwin_version="$(uname -r)"
+if [ "${darwin_version%%.*}" -ge 19 ]; then
+    if ! [ -d "/nix" ]; then
+        (echo 'nix'; echo 'run	private/var/run') | sudo tee -a /etc/synthetic.conf >/dev/null
+        echo "Added /nix and /run to synthetic.conf. You must now reboot and re-run this script."
+        exit 0
+    fi
+
+    # If 10.15 and nothing mounted on /nix
+    if ! mount | grep -q 'on /nix'; then
```

Comment:
Will this work on non-English locales?

Reply:
took your advice and added `LANG=en_US` (ew!)